mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -250,44 +250,40 @@ def _load_mesh(self):  # {{{
             refTopDepth, refLayerThickness
         # }}}
 
-    def _cache_velocity_climatologies(self):  # {{{
+    def _compute_velocity_climatologies(self):  # {{{
         '''compute yearly velocity climatologies and cache them'''
 
-        variableList = ['avgNormalVelocity',
-                        'avgVertVelocityTop']
```

Comment:
yes, that is the function I was thinking in my first comment (here and on the main Conversation page). But I agree that it would be a lot of clutter.